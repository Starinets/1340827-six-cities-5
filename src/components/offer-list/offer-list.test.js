import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import OfferList from './offer-list';
import {
  emptyFunction,
  OfferPlace,
  Offers
} from '../../test-data/test-data';

describe(`Component rendered correctly -> OfferList`, () => {
  it(`on Main page`, () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
        <OfferList
          offers={ Offers }
          offerPlace={ OfferPlace.CITIES }
          changeHoveredOffer={ emptyFunction }
        />
    );
    expect(tree).toMatchSnapshot();
  });

  it(`on Offer page, as neighborhood list`, () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
        <OfferList
          offers={ Offers }
          offerPlace={ OfferPlace.NEIGHBORHOOD }
          changeHoveredOffer={ emptyFunction }
        />
    );
    expect(tree).toMatchSnapshot();
  });

  it(`on Favorites page`, () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
        <OfferList
          offers={ Offers }
          offerPlace={ OfferPlace.FAVORITES }
          changeHoveredOffer={ emptyFunction }
        />
    );
    expect(tree).toMatchSnapshot();
  });
});
