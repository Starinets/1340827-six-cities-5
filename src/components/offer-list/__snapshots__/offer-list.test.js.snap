// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component rendered correctly -> OfferList on Favorites page 1`] = `
<div
  className="near-places__list places__list"
>
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 4,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.938361,
            "longitude": 6.959974,
            "zoom": 13,
          },
          "name": "Cologne",
        },
        "description": "Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.",
        "features": Array [
          "Laptop friendly workspace",
          "Washer",
          "Towels",
          "Fridge",
          "Air conditioning",
          "Baby seat",
          "Breakfast",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 1,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.957361,
          "longitude": 6.9509739999999995,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 493,
        "rating": 3.8,
        "type": "house",
      }
    }
    offerPlace="favorites__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 8,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
        "features": Array [
          "Towels",
          "Baby seat",
          "Laptop friendly workspace",
          "Dishwasher",
          "Washing machine",
          "Air conditioning",
          "Washer",
          "Coffee machine",
          "Breakfast",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 2,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.839557,
          "longitude": 4.346697,
          "zoom": 16,
        },
        "name": "Amazing and Extremely Central Flat",
        "price": 323,
        "rating": 4.3,
        "type": "apartment",
      }
    }
    offerPlace="favorites__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 4,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Washer",
          "Laptop friendly workspace",
          "Breakfast",
          "Air conditioning",
          "Baby seat",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 3,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.865556999999995,
          "longitude": 4.371696999999999,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 218,
        "rating": 4.6,
        "type": "apartment",
      }
    }
    offerPlace="favorites__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 52.37454,
            "longitude": 4.897976,
            "zoom": 13,
          },
          "name": "Amsterdam",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Dishwasher",
          "Air conditioning",
          "Laptop friendly workspace",
          "Washer",
          "Baby seat",
          "Towels",
          "Breakfast",
          "Coffee machine",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 4,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 52.385540000000006,
          "longitude": 4.902976,
          "zoom": 16,
        },
        "name": "Perfectly located Castro",
        "price": 440,
        "rating": 4.6,
        "type": "house",
      }
    }
    offerPlace="favorites__"
    onMouseOver={[Function]}
  />
</div>
`;

exports[`Component rendered correctly -> OfferList on Main page 1`] = `
<div
  className="cities__places-list places__list tabs__content"
>
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 4,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.938361,
            "longitude": 6.959974,
            "zoom": 13,
          },
          "name": "Cologne",
        },
        "description": "Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.",
        "features": Array [
          "Laptop friendly workspace",
          "Washer",
          "Towels",
          "Fridge",
          "Air conditioning",
          "Baby seat",
          "Breakfast",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 1,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.957361,
          "longitude": 6.9509739999999995,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 493,
        "rating": 3.8,
        "type": "house",
      }
    }
    offerPlace="cities__place-"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 8,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
        "features": Array [
          "Towels",
          "Baby seat",
          "Laptop friendly workspace",
          "Dishwasher",
          "Washing machine",
          "Air conditioning",
          "Washer",
          "Coffee machine",
          "Breakfast",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 2,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.839557,
          "longitude": 4.346697,
          "zoom": 16,
        },
        "name": "Amazing and Extremely Central Flat",
        "price": 323,
        "rating": 4.3,
        "type": "apartment",
      }
    }
    offerPlace="cities__place-"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 4,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Washer",
          "Laptop friendly workspace",
          "Breakfast",
          "Air conditioning",
          "Baby seat",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 3,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.865556999999995,
          "longitude": 4.371696999999999,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 218,
        "rating": 4.6,
        "type": "apartment",
      }
    }
    offerPlace="cities__place-"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 52.37454,
            "longitude": 4.897976,
            "zoom": 13,
          },
          "name": "Amsterdam",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Dishwasher",
          "Air conditioning",
          "Laptop friendly workspace",
          "Washer",
          "Baby seat",
          "Towels",
          "Breakfast",
          "Coffee machine",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 4,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 52.385540000000006,
          "longitude": 4.902976,
          "zoom": 16,
        },
        "name": "Perfectly located Castro",
        "price": 440,
        "rating": 4.6,
        "type": "house",
      }
    }
    offerPlace="cities__place-"
    onMouseOver={[Function]}
  />
</div>
`;

exports[`Component rendered correctly -> OfferList on Offer page, as neighborhood list 1`] = `
<div
  className="near-places__list places__list"
>
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 4,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.938361,
            "longitude": 6.959974,
            "zoom": 13,
          },
          "name": "Cologne",
        },
        "description": "Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.",
        "features": Array [
          "Laptop friendly workspace",
          "Washer",
          "Towels",
          "Fridge",
          "Air conditioning",
          "Baby seat",
          "Breakfast",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 1,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.957361,
          "longitude": 6.9509739999999995,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 493,
        "rating": 3.8,
        "type": "house",
      }
    }
    offerPlace="near-places__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 8,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
        "features": Array [
          "Towels",
          "Baby seat",
          "Laptop friendly workspace",
          "Dishwasher",
          "Washing machine",
          "Air conditioning",
          "Washer",
          "Coffee machine",
          "Breakfast",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 2,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.839557,
          "longitude": 4.346697,
          "zoom": 16,
        },
        "name": "Amazing and Extremely Central Flat",
        "price": 323,
        "rating": 4.3,
        "type": "apartment",
      }
    }
    offerPlace="near-places__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 4,
        "city": Object {
          "location": Object {
            "latitude": 50.846557,
            "longitude": 4.351697,
            "zoom": 13,
          },
          "name": "Brussels",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Washer",
          "Laptop friendly workspace",
          "Breakfast",
          "Air conditioning",
          "Baby seat",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 3,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 50.865556999999995,
          "longitude": 4.371696999999999,
          "zoom": 16,
        },
        "name": "The house among olive ",
        "price": 218,
        "rating": 4.6,
        "type": "apartment",
      }
    }
    offerPlace="near-places__"
    onMouseOver={[Function]}
  />
  <OfferCard
    imageSize={
      Object {
        "HEIGHT": 200,
        "WIDTH": 260,
      }
    }
    offer={
      Object {
        "adultsCount": 7,
        "bedroomsCount": 3,
        "city": Object {
          "location": Object {
            "latitude": 52.37454,
            "longitude": 4.897976,
            "zoom": 13,
          },
          "name": "Amsterdam",
        },
        "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
        "features": Array [
          "Dishwasher",
          "Air conditioning",
          "Laptop friendly workspace",
          "Washer",
          "Baby seat",
          "Towels",
          "Breakfast",
          "Coffee machine",
          "Fridge",
        ],
        "host": Object {
          "avatar": "img/avatar-angelina.jpg",
          "id": 25,
          "isPro": true,
          "name": "Angelina",
        },
        "id": 4,
        "image": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
        "images": Array [
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
          "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
        ],
        "isFavorite": false,
        "isPremium": false,
        "location": Object {
          "latitude": 52.385540000000006,
          "longitude": 4.902976,
          "zoom": 16,
        },
        "name": "Perfectly located Castro",
        "price": 440,
        "rating": 4.6,
        "type": "house",
      }
    }
    offerPlace="near-places__"
    onMouseOver={[Function]}
  />
</div>
`;
